bitrise.yml

```diff
@@ -15,11 +15,12 @@ pipelines:
               expression: '{{ enveq "BUILD_FOCUS_IOS" "true" }}'
             depends_on:
             - determine_apps_affected
-        klar_configure_build:
-            run_if:
-              expression: '{{ enveq "BUILD_FOCUS_IOS" "true" }}'
-            depends_on:
-            - determine_apps_affected
+        # TODO: May belong to pipeline_build_and_test_focus instead
```

Comment:
Could we remove klar_configure_build altogether?

Reply:
Let's keep it here for now

Reply:
@clarmso I am actually thinking that if we have the env variable to false, we skip this as we do for focus and there is no need to comment out, this will not be executed